modular_skyrat/modules/sec_haul/code/misc/packs.dm

```diff
@@ -38,12 +38,14 @@
 	desc = "A classic Shotgun used by hunters, police and frontiersmen alike, now at an affordable price."
 	cost = PAYCHECK_COMMAND * 20
 	contains = list(/obj/item/gun/ballistic/shotgun/riot, /obj/item/storage/pouch/ammo, /obj/item/storage/belt/bandolier, /obj/item/ammo_box/advanced/s12gauge/hunter)
+	access_view = ACCESS_WEAPONS
 
 /datum/supply_pack/goody/rensoter_shotgun
 	name = "Renoster Shotgun Single-Pack"
 	desc = "A common modern shotgun used by Solarian State Police."
 	contains = list(/obj/item/gun/ballistic/shotgun/riot/sol, /obj/item/storage/pouch/ammo, /obj/item/storage/belt/bandolier, /obj/item/ammo_box/advanced/s12gauge/rubber)
 	cost = PAYCHECK_COMMAND * 22
+	access_view = ACCESS_WEAPONS
 
 /datum/supply_pack/goody/double_barrel
```

Comment:
The double shotty too

Reply:
It already is a TG weapon thats permit locked DUMMY

Reply:
Can you just get this overwith already